bag-ui/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="label_bag">BAG</string>
+    <string name="label_color">Color:</string>
+    <string name="label_size">Size:</string>
+    <string name="label_zero">0</string>
+    <string name="label_check_out">CHECK OUT</string>
```

Comment:
It isn't a good idea to create caps-locked text in strings.xml file. You can create the string `<string name="label_check_out">Check out</string> and in places where you need it to be caps-locked there is usually exist attribute `android:textAllCaps = true`.